packages/material-ui/src/Chip/Chip.js

```diff
@@ -130,7 +130,7 @@ export const styles = (theme) => {
       border: `1px solid ${
         theme.palette.type === 'light' ? 'rgba(0, 0, 0, 0.23)' : 'rgba(255, 255, 255, 0.23)'
       }`,
-      '$clickable&:hover, $clickable&:focus, $deletable&:focus': {
+      '&$focusVisible, $clickable&:hover': {
```

Comment:
We lose the style on deletable chip only. Probably my fault when I have removed the focus visible hook to anticipate the change of structure in 

x3

I think that we can solve this by using ButtonBase

Reply:
When I tested the chips using the demo the focus styling was applied to deletable ones as well?

Reply:
You can test the live preview, the style isn't applied

Reply:
You are right. 6808f9e should fix that 👍